### PR TITLE
Details filtering

### PR DESCRIPTION
**Pull request by ujjwalvkulkarni**
Definition of Done:

1. Users can filter reviews by courses on Professor detail page.
2. Users can filter reviews by professor on Course detail page.

### CODE REVIEW

File: ratenyu/courses/templates/courses/detail.html [hunk 1]
```diff
@@ -38,13 +38,14 @@ <h2>{{ reviews_list|length }} Reviews</h2>
...
-        <li class="detail-side-links"><a href="">{{ professor.name }}</a></li>
+            <li class="detail-side-links"><a href='#' onclick = "professorSelect('{{professor.name}}')">{{ professor.name }}</a></li>
         {% endfor %}
+        <li class="detail-side-links"><a href='#' onclick = "professorSelect('{{professors_list.name}}')">All</a></li>
     </ul>
 
     <div class="all-reviews">
         {% for review in reviews_list %}
-        <div class="review">
+        <div class="review" review_professor = "{{review.professor_obj.name}}">
```
Comment (virenmparmar) [inline ratenyu/courses/templates/courses/detail.html:48]: I feel like you don't need this. Can we use the professor name that is mentioned in detail-side-links class tag.

File: ratenyu/courses/templates/courses/detail.html [hunk 2]
```diff
@@ -38,13 +38,14 @@ <h2>{{ reviews_list|length }} Reviews</h2>
     <ul class="detail-side-links">
         <li class="detail-side-links"><span class="rev-title">Professor:</span></li>
         {% for professor in professors_list %}
-        <li class="detail-side-links"><a href="">{{ professor.name }}</a></li>
+            <li class="detail-side-links"><a href='#' onclick = "professorSelect('{{professor.name}}')">{{ professor.name }}</a></li>
```
Comment (virenmparmar) [inline ratenyu/courses/templates/courses/detail.html:41]: href seems like it is refreshing the page. do we need this? Can't we change the css styling to look like it is a link?

**Review by virenmparmar — COMMENTED** [review]
Overall, it LGTM.
I have a couple of doubts, i have added inline comments. Consider those points for poth the details pages

File: ratenyu/courses/templates/courses/detail.html (hunk 2 above)
Comment (ujjwalvkulkarni) [reply ratenyu/courses/templates/courses/detail.html:41]: href='#' stops the page from refreshing but also keeps it as a link. Is it refreshing on your browser?

**Review by virenmparmar — APPROVED** [review]
LGTM

**Pull request merged**